src/Esfa.Vacancy.Register.Application/Queries/SearchApprenticeshipVacancies/VacancySearchParametersBuilder.cs

```diff
@@ -39,7 +39,8 @@ public VacancySearchParametersBuilder(IStandardCodeConverter standardCodeConvert
             {
                 PageNumber = request.PageNumber,
                 PageSize = request.PageSize,
-                SubCategoryCodes = combinedSubCategoryCodes
+                SubCategoryCodes = combinedSubCategoryCodes,
+                PostedInDays = request.PostedInDays
```

Comment:
You are passing in the int but only using the DateTime in the query. Any reason you don't just do the conversion in the params builder? I can see that you've tested that the conversion happens.

Reply:
yes could do that.

Reply:
fixed